flink-clients/src/main/java/org/apache/flink/client/program/PackagedProgramUtils.java

```diff
@@ -44,13 +45,15 @@
 	 * @param packagedProgram to extract the JobGraph from
```

Comment:
Maybe add to the JavaDoc, that it generates a `JobGraph` with the specified `JobID`.